### PR TITLE
chore(wallet): some minor cleanups, move sheets to appropriate level,…

### PR DESCRIPTION
**Pull request by J-Son89**
This pr is just a few small refactors.

I have:
moved the bottom sheets files in the wallet from `status-im.contexts.wallet.common.sheets.*` to `status-im.contexts.wallet.sheets.*`
This should make no difference to anything and will not need to be test.

removed the rf event `:wallet/select-send-account-address` as it is unused. - so this won't have any effect here.

I also made a minor adjustment to the api of the rf event `wallet/select-send-address` to read a bit easier.
This could do with a happy path testing of the send flows for an asset and a token, however the changes are quite small and should be quite "safe"

### CODE REVIEW

File: src/status_im/contexts/wallet/send/events.cljs [hunk 1]
```diff
@@ -47,13 +47,6 @@
             (update-in [:wallet :ui :send] dissoc :route)
             (update-in [:wallet :ui :send] dissoc :loading-suggested-routes?))}))
 
-(rf/reg-event-fx :wallet/select-send-account-address
```
Comment (J-Son89) [inline src/status_im/contexts/wallet/send/events.cljs:48]: this event was unused

File: src/status_im/contexts/wallet/send/events.cljs [hunk 2]
```diff
@@ -47,20 +47,13 @@
...
-            (update-in [:wallet :ui :send] dissoc :to-address))
-    :fx [[:dispatch [:navigate-to-within-stack [:wallet-select-asset stack-id]]]]}))
-
 (rf/reg-event-fx :wallet/clean-send-address
  (fn [{:keys [db]}]
    {:db (update-in db [:wallet :ui :send] dissoc :recipient :to-address)}))
 
 (rf/reg-event-fx
  :wallet/select-send-address
- (fn [{:keys [db]} [{:keys [address token recipient stack-id]}]]
```
Comment (J-Son89) [inline src/status_im/contexts/wallet/send/events.cljs:61]: a minor detail to this api to make it more readable, previously it looked as though we are using all of the token details in this endpoint

**Review by ulisesmac — APPROVED** [review]
Thanks for this PR! :+1:  :+1:

**Review by smohamedjavid — APPROVED** [review]

**Review by OmarBasem — APPROVED** [review]

**Comment by VolodLytvynenko** [comment]
hey @J-Son89 could you please rebase the current PR and resolve conflicts? thank you

**Comment by J-Son89** [comment]
@VolodLytvynenko - done!

**Comment by VolodLytvynenko** [comment]
Hi @J-Son89 thank you for PR. no issues from my side. Ready to be merged

**Pull request merged**